server/src/main/kotlin/org/itmodreamteam/myrest/server/service/notification/FakeNotificationService.kt

```diff
@@ -0,0 +1,15 @@
+package org.itmodreamteam.myrest.server.service.notification
+
+import org.itmodreamteam.myrest.server.model.user.User
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+class FakeNotificationService : NotificationService {
+
+    private val log = LoggerFactory.getLogger(javaClass)
+
+    override fun notify(user: User, text: String) {
```

Comment:
We really gonna pass User entity? We can use some of user dto's

Reply:
This method needs to know which user to notify. All we need to determine a user is its ID.
Here we pass User so as not to fetch it from DB inside this method once more.
If it is convenient, we can add `fun notify(userId: Long, text: String)` any time later.